HelloWorld/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>  
+using namespace std;  
+
+//Source: https://www.javatpoint.com/prime-number-program-in-cpp
+
+int isPrime(int n)
+{
+int i, m=0;
```

Comment:
Split out and simplify initialization. Don't declare an iterator variable until it's needed
```suggestion
int m = n / 2;
```